packages/@angular/cli/lib/config/schema.json

```diff
@@ -292,6 +292,10 @@
       },
       "additionalProperties": false
     },
+    "packageManager": {
+        "enum": [ "npm", "cnpm", "yarn" ],
```

Comment:
I would like the default to be a fourth value, so that people who forces it to `npm` don't get the message.